polkadot/xcm/pallet-xcm-benchmarks/src/generic/benchmarking.rs

```diff
@@ -113,6 +113,19 @@ benchmarks! {
 		executor.bench_process(xcm)?;
 	} verify {}
 
+	set_asset_claimer {
+		let mut executor = new_executor::<T>(Default::default());
+		let (sender_account, sender_location) = account_and_location::<T>(1);
```

Comment:
```suggestion
		let (_, sender_location) = account_and_location::<T>(1);
```

If you don't use this variable clippy will complain, you can run `cargo clippy -p pallet-xcm-benchmarks --all-features` to see what it says